main.go

```diff
@@ -68,12 +74,19 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	var envVars EnvVars
+
 	namespace, err := getNamespace()
 	if err != nil {
 		setupLog.Error(err, "unable to get namespace name")
 		os.Exit(1)
 	}
 
+	if err = getEnvVars(&envVars); err != nil {
```

Comment:
Can we just return the envVars from the function, I do not see the need to send a struct to the function

Reply:
done.